js/build.sh

```diff
@@ -10,7 +10,7 @@ mkdir -p dist
 # copy the rive wasm js and type definitions files to build dir
 if [ ! -f ../wasm/publish/rive.mjs ]; then 
     pushd ../wasm
-    ./build-all-wasm.sh
+    ./build_all_wasm.sh
```

Comment:
This is a little pedantic, but in every other repo we take care to use _ instead of - in filenames, so I thought I'd normalize here...